Automattic/wp-calypso
# Jetpack Products: Add new constants for realtime products

#### Changes proposed in this Pull Request

This PR adds constants for the upcoming realtime Jetpack products. 

#### Testing instructions



1. Verify that new product constants are added.
2. Verify that old product constants are still there.
3. Verify that the pricing page has not changed.



Related to 1200412004370260-as-1200549781126143
